.travis.yml

```diff
@@ -3,9 +3,9 @@ language: java
 matrix:
   include:
     - os: linux
-      jdk: oraclejdk8
+      jdk: oraclejdk9
     - os: linux
-      jdk: openjdk7
+      jdk: openjdk9
 
```

Comment:
Should we keep open/oraclejdk8 for a while to ensure backwards compatability?

Reply:
It unfortunately isn't working. jdk_switcher no longer works with the version of Linux that is being installed if I recall correctly. At this point, figured it was just more important to have our tests actually run and work.